src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
+
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/movie';
+
+type Props = {
+  movie: Movie[];
+};
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+export const MoviesList: React.FC<Props> = ({ movie }) => (
+  <>
+    <div className="movies">
+      {movie.map(item => (
+        <MovieCard
+          title={item.title}
+          description={item.description}
+          imgUrl={item.imgUrl}
+          imdbUrl={item.imdbUrl}
+          imdbId={item.imdbId}
+        />
+      ))}
```

Comment:
provide whole card here, not each property separatly
```suggestion
      {movie.map(item => (
        <MovieCard
          card={item}
        />
      ))}
```

and don't forget about key=""

Reply:
Comments fixed.
Sorry but I don't understand your comment regarding demo link. I put it to the first pull request comment and to the README.